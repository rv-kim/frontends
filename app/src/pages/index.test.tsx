import * as React from 'react'
import { shallow } from 'enzyme'
import * as ShallowRenderer from 'react-test-renderer/shallow'
// import configureStore from 'redux-mock-store'
import { Index } from './index'

const renderer = new ShallowRenderer();
// const mockStore = configureStore()

describe('Index Component', () => {

  test('snapshots_01', () => {
    const ss = renderer.render(<Index />);
    expect(ss).toMatchSnapshot()
  })

  test('test 01', () => {
    // const store = mockStore({count: 0})
    // const index = shallow(<Index store={store} />)
    const index = shallow(<Index />)
    expect(index).toEqual(index)
  })
})
