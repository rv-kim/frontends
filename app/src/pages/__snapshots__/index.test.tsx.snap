// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Component snapshots_01 1`] = `
<div>
  <Link
    href="/test"
  >
    <a>
      link to test
    </a>
  </Link>
  <hr />
  <Sample
    color="green"
    text="good, bye"
  />
  <hr />
  <Connect(Counter) />
</div>
`;
